src/main/java/com/cburch/logisim/file/XmlWriter.java

```diff
@@ -96,32 +98,20 @@ static String attrsToString(NamedNodeMap a) {
     return String.join(" ", lst);
   }
 
-  static int stringCompare(String a, String b) {
-    if (a == null) return -1;
-    if (b == null) return 1;
-    return a.compareTo(b);
+  private static final int stringCompare(String string1, String string2) {
+    if (string1 == null) return -1;
```

Comment:
Can we have `stringA`, 'stringB` not `string1`, `stringB` for consistency? Because we use `a`, `b` later.

Reply:
Done